### DISCUSSION TITLE
How to avoid this gaps inside walls?

### DISCUSSION THREAD

**Discussion by mmiguel4** [supermerill/SuperSlicer#2309] [Q&A] [answered]
In some walls this gaps are not filled, how to avoid this? T
![gap](https://user-images.githubusercontent.com/30552855/152335613-37607084-4a70-435e-9ab5-59d6f2626037.png)

Thank you

**Comment by supermerill** [comment, marked as answer]
enable gapfill, or overlapping perimeters (all)

**Comment by mmiguel4** [comment]
Overlapping "Also for all perimeters" option did the trick.

Thank you
